### ISSUE TITLE
The attachment '<filename>' can not be removed, the storage name is unknown

### ISSUE THREAD

**Issue by Memphizzz** [Sicos1977/MSGReader#79]
I'm trying to delete the attachments of a mail using a foreach like this:
```
 foreach (var attachment in mail.Attachments)
        mail.DeleteAttachment(attachment);
```

But I get the following error:
`MsgReader.Exceptions.MRCannotRemoveAttachment: 'The attachment '<filename>' can not be removed, the storage name is unknown'`

It looks like this error is coming from: MsgReader.Outlook.Storage.Message:2604.

This is the first email (msg file) that behaves like this.
Do you have any idea what might cause the attachment to not have a "StorageName"? 

P.S.: Due to the NuGet package issue, this was using version 2.0.14 but it also happens on 2.1.0.

**Comment by Sicos1977** [comment]
I released a new version 2.1.1 can you check if this fixes your error?

**Comment by Memphizzz** [comment]
Fixed! Thank you!
